src/motor_node.cc

```diff
@@ -443,6 +546,12 @@ int main(int argc, char* argv[]) {
                 robot->setWheelType(robot->wheel_type);
                 mcbStatusPeriodSec.sleep();
             }
+            // a periodic refresh of drive type which is a safety net due to it's importance.
```

Comment:
Just in case the robot suddenly grows two extra wheels 😛

Reply:
This is sort of a just in case for something I had seen but cannot figure out where this was not set and all heck broke loose.